src/open-in-atom.ts

```diff
@@ -1,87 +1,60 @@
 import * as sourcegraph from 'sourcegraph'
 import * as path from 'path'
 
-const supportedEditors: {[editor: string]: {urlPattern: string}} = {
-    vscode: {
-        urlPattern: 'vscode://file%file:%line:%col',
-    },
-    idea: {
-        urlPattern: 'idea://open?file=%file&line=%line&column=%col',
-    },
-    sublime: {
-        urlPattern: 'subl://open?url=%file&line=%line&column=%col',
-    },
-    custom: {
-        urlPattern: '',
-    },
+interface Settings {
+    'openinatom.basePath'?: string
```

Comment:
**nitpick**: Settings are camelCase, so can we rename this to `openInAtom.basePath`?